Backend/Backend/Models/Bookmark.cs

```diff
@@ -1,24 +1,27 @@
 ﻿using System.ComponentModel.DataAnnotations;
+using Backend.Infrastructure;
 
 namespace Backend.Models;
 
 public class Bookmark : BaseEntity
 {
+    private readonly AppDbContext _db;
 
-    public Bookmark() 
+    public Bookmark(AppDbContext db) 
     {
         Title = string.Empty;
         Note = string.Empty;
+        _db = db;
     }
 
     [Key]
     public Guid Id { get; set; }
 
-    public User User { get; set; } = null!;
+    public User? User => _db.Users.FirstOrDefault(u => u.Id == UserId);
```

Comment:
Instead of doing a linq query in the model itself, use an include() in the service to get the user.

Reply:
hmmm. I don't get what you mean by that

Reply:
![Capture](https://github.com/awaazo/awaazo/assets/46968018/979e8ad0-e251-42e5-b802-58087622f197)

Reply:
If you add an include() to your linq query, it will automatically get the user by Id.